distance3d/gjk/_gjk_jolt.py

```diff
@@ -210,7 +214,9 @@ def gjk_distance_jolt(
             # Get the closest points
             a, b = calculate_closest_points(Y, P, Q, n_points)
 
-            assert abs(np.dot(search_direction, search_direction) - v_len_sq) < 1e-12
+            check_value = abs(np.dot(search_direction, search_direction) - v_len_sq)
+            assert not sanity_check or (sanity_check and check_value < sanity_check), f"Sanity check failed: {check_value}"
```

Comment:
If it can be used I'd suggest to document this. However, I'd like to avoid this. In case you don't want to have a sanity check, you can just set the value to infinity.

Reply:
Do you want the assertion to be rewritten or adapt the docstring - or both ?

Reply:
I prefer rewriting the assert.